.gitpod.Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM gitpod/workspace-full:latest
```

Comment:
:wrench:  dont use `latest`, pin to version and have dependabot reminf of updates.

Reply:
Yep, that's the change I'm going to make to test out the branch protection rules.